docs/building_models.md

```diff
@@ -64,12 +64,12 @@ class HeadAndNeck(HeadAndNeckBase):
     def __init__(self) -> None:
         super().__init__(
             ground_truth_ids=["parotid_l", "parotid_r", "smg_l", "smg_r", "spinal_cord"]
-            azure_dataset_id="id-of-your-blob-containing-prostate-data")
+            azure_dataset_id="name-of-your-AML-dataset-with-prostate-data")
 ```
 
 ### Training a new model
 
-* Set up your model configuration as above.
+* Set up your model configuration as above and update `azure_dataset_id` to the name of your Dataset in the AML workspace.
```

Comment:
Add: It is enough to put your dataset into blob storage. The dataset should be a contained in a folder at the root of the `datasets` container. The InnerEye runner will check if there is a dataset in the AzureML workspace already, and if not, generate it directly from blob storage.